### PR TITLE
Issue #6207: Add Xpath Regression Test for PackageName

### PR DESCRIPTION
**Pull request by TanayMorakhia**
part of #6207

### CODE REVIEW

**Comment by TanayMorakhia** [comment]
I think the test fail is not because of this PR.

File: src/it/resources/org/checkstyle/suppressionxpathfilter/packagename/SuppressionXpathRegressionPackageNameTwo.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package org.checkstyle.suppressionxpathfilter.packagename; // warn
```
Comment (romani) [inline src/it/resources/org/checkstyle/suppressionxpathfilter/packagename/SuppressionXpathRegressionPackageNameTwo.java:1]: Same as in Two.
Please make package name wrapped in few lines, to make it a bit different.
And please add comment before `package`

**Review by romani — APPROVED** [review]
Items

**Review by romani — CHANGES_REQUESTED** [review]
See above

File: src/it/resources/org/checkstyle/suppressionxpathfilter/packagename/SuppressionXpathRegressionPackageNameTwo.java (hunk 1 above)
Comment (TanayMorakhia) [reply src/it/resources/org/checkstyle/suppressionxpathfilter/packagename/SuppressionXpathRegressionPackageNameTwo.java]: done

**Review by romani — APPROVED** [review]
Ok to merge

**Pull request merged**